frontend/packages/dev-console/src/components/health-checks/AddHealthChecks.tsx

```diff
@@ -59,6 +60,10 @@ const AddHealthChecks: React.FC<FormikProps<FormikValues> & AddHealthChecksProps
   const resourceKind = modelFor(kindForCRDResource).crd ? kindForCRDResource : kind;
   const isFormClean = _.every(values.healthChecks, { modified: false });
 
+  const healthLink = isUpstream()
+    ? `${openshiftHelpBase}applications/application-health.html`
+    : `${openshiftHelpBase}html/building_applications/application-health`;
```

Comment:
We have links to the OSD docs as well. Should those use the `html/building_applications/application-health` path?

Reply:
There is no equivalent page in the OSD docs afaics, so this must have been pointing to OCP docs even when run in OSD.  Or did I misunderstand you here?